scheduler/cron.go

```diff
@@ -0,0 +1,198 @@
+package scheduler
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os/exec"
+	"strings"
+)
+
+type cron struct{}
+
+// Enable writes Fluidkeys' cron lines into crontab
+func (c *cron) Enable() (crontabWasAdded bool, err error) {
+	crontab := &systemCrontab{}
+	crontabWasAdded, err = c.enable(crontab)
+	if err != nil {
```

Comment:
When we discussed it, `Enable` did exactly 1 thing:

```
return c.enable(&systemCrontab{})
```

By adding more stuff (wrapping the error check on `enable`), `Enable` is no longer trivial: it has its own behaviour.`

Reply:
Yep, stripped back.